src/Skeletor/Manager/PackageManager.php

```diff
@@ -163,6 +163,46 @@ public function configure(Package $package, Framework $activeFramework)
     {
         if (!$this->options['dryRun']) {
             $package->configure($activeFramework);
+
+            // Update .env files
+            $this->updateEnvironmentVariables($package);
+        }
+    }
+
+    /**
+     * @param  Package $package
+     */
+    public function publishConfig(Package $package)
+    {
+        if (!$this->options['dryRun']) {
+            $output = $package->publishConfig();
+
+            $this->cli->br()->output('Publishing configuration');
+            $this->cli->output($output);
+        }
+    }
+
+    /**
+     * Update the .env and .env.example files with the environment variables
+     * that are specified for the package.
+     *
+     * @param  Package $package
+     */
+    private function updateEnvironmentVariables(Package $package)
+    {
+        // Write the package environment variables to the .env files
+        if ($package->hasEnvironmentVariables() === true) {
```

Comment:
Not sure we had this conversation before, my personal preference is to drop === true, seems redundant since the method name starting with "has" already gives away that a boolean will be returned. If we decide to leave it, I find it more readable the other way around, true === VALUE. It comes down to personal preference, as long as we are consistence, all good with me.